dev-infrastructure/docs/development-setup.md

```diff
@@ -333,13 +333,13 @@ Then register it with the Maestro Server
 2) Create pre-requisite resources for cluster creation
 
     Replace `resource-group`, `vnet-name`, `nsg-name` and `subnet-name` with any valid names.
-  
+
   - Create a resource group for your ARO HCP cluster. This is used, alongside the resource name and subscription ID, to represent
-    your ARO HCP cluster resource in Azure. 
+    your ARO HCP cluster resource in Azure.
     ```
     az group create --name <resource-group> --location "westus3"
     ```
-  - Create a Virtual Network. 
+  - Create a Virtual Network.
```

Comment:
Not sure why these spaces were added but the additional backtick above is useful